crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -14,12 +14,16 @@ use opentelemetry_sdk::{
     trace, Resource,
 };
 use opentelemetry_semantic_conventions::resource;
+use settings::{ConsoleLogSettings, OtlpLogSettings, OtlpTraceSettings};
 use snafu::{ResultExt as _, Snafu};
-use tracing::{level_filters::LevelFilter, subscriber::SetGlobalDefaultError};
+use tracing::subscriber::SetGlobalDefaultError;
 use tracing_subscriber::{filter::Directive, layer::SubscriberExt, EnvFilter, Layer, Registry};
 
+pub mod settings;
+
 type Result<T, E = Error> = std::result::Result<T, E>;
 
+#[allow(missing_docs)]
```

Comment:
nit: I think we _currently_ don't need that attribute.

Reply:
Done in afc21636bf3ec0324aabd3dcf20b65dfe38de946